### DISCUSSION TITLE
How to enable "Simple" theme when running locally?

### DISCUSSION THREAD

**Discussion by sabinayakc** [jgraph/drawio#4060] [Q&A] [answered]
Hello, I cloned the repo, built a docker image but I can't see the "Simple" and "Atlas" themes. I see that there is a reference to `js/diagramly/Simple.js` but the file is not there?

How can I use the simple theme when running locally?

**Comment by davidjgraph** [comment, marked as answer]
The simple and atlas themes are no longer supported by us. We just didn't use them.

**Reply by sabinayakc** [reply]
Is it going to be removed in the next release? Since I can still use it via the online editor and also the latest docker image has those options.

**Reply by davidjgraph** [reply]
It will be removed at some point. Do not rely on the functionality, use the standard theme instead.

**Reply by sabinayakc** [reply]
Okay thank you. Closing this discussion.

**Comment by davidjgraph** [comment]
The simple theme is available again in the build.
